Bottom: 8d4366cc907993666970d4d3cd7ad6f10cbde8d7
Top:    b1a43be6ff8dbdfe6dead856d82578df43d66f38
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:21:37 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ddcf35e..0a264ce 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -239,7 +239,7 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
 }
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   FILE *f;
   time_t currenttime;
diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index 4f40a3b..b967543 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -9,5 +9,5 @@ object3d *make_object3d (void);
 void destroy_object3d (object3d *object);
 object3d *object3d_create_test_cube (void);
 object3d *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
 void object3d_test_board_outline (void);
